web/src/lib/domain-logic/allowFormation.ts

```diff
@@ -10,15 +10,7 @@ export const allowFormation = (
   legalStructureId: string | undefined,
   persona: BusinessPersona | undefined
 ): boolean => {
-  const featureFlagMap: Partial<Record<FormationLegalType, boolean>> = {
-    "foreign-limited-liability-company": process.env.FEATURE_BUSINESS_FLC === "true",
-    "foreign-limited-liability-partnership": process.env.FEATURE_BUSINESS_FLLP === "true",
-    "foreign-limited-partnership": process.env.FEATURE_BUSINESS_FLP === "true",
-    "foreign-s-corporation": process.env.FEATURE_BUSINESS_FCORP === "true",
-    "foreign-c-corporation": process.env.FEATURE_BUSINESS_FCORP === "true",
-    "foreign-nonprofit": process.env.FEATURE_BUSINESS_FNP === "true",
-    nonprofit: process.env.FEATURE_BUSINESS_NP === "true",
-  };
+  const featureFlagMap: Partial<Record<FormationLegalType, boolean>> = {};
```

Comment:
[sand] We don't need this

Reply:
Oof, this caused a lot of refactoring. I originally left it in here so it was easy for people to add feature flags again, but now I've simplified this file not to depend on the feature flags list.